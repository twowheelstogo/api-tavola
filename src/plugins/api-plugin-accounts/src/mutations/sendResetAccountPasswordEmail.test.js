import mockContext from "@reactioncommerce/api-utils/tests/mockContext.js";
import sendResetAccountPasswordEmail from "./sendResetAccountPasswordEmail.js";

mockContext.mutations.sendResetAccountPasswordEmail = jest.fn().mockName("mutations.sendResetAccountPasswordEmail");

test("throws if account matching email not found", async () => {
  mockContext.collections.Accounts.findOne.mockReturnValueOnce(undefined);
  await expect(sendResetAccountPasswordEmail(mockContext, { email: "test@email.com", url: "https://example.com" })).rejects.toThrowErrorMatchingSnapshot();
});
