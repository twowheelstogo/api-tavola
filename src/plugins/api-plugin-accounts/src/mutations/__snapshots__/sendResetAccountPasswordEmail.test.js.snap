// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throws if account matching email not found 1`] = `"Account not found"`;
